COMMUNITY.md

```diff
@@ -13,13 +13,40 @@ As opposed to defining a big overarching burden of responsibilities the team wou
   - Slack
 - Oversight of [this repository](https://github.com/fluxcd/community).
 
+### Communications
+
+We as a project want to make it a priority to inform our community and the wider world of developments well. It's what helps new contributors get up to speed, enables collaboration, integrations and more.
+
+Part of our team wants to own comms on
+
+- Slack: [#flux](https://cloud-native.slack.com/archives/flux), [#gitops](https://kubernetes.slack.com/archives/gitops), likely more to follow
+- the `cncf-flux-dev` [project mailing list](https://lists.cncf.io/g/cncf-flux-dev)
+- [our blog](https://github.com/fluxcd/website/issues/27) (WIP)
+- Twitter (WIP)
+- [LinkedIn](https://www.linkedin.com/groups/8985374/)
+- [Kubernetes Discuss - Ecosystem forum](https://discuss.kubernetes.io/c/announcements/announcements-eco)
```

Comment:
The list looks a bit "messy" due to some items starting with a capital letter, and others don't. Maybe this is a bit better?

```suggestion
- The `cncf-flux-dev` [project mailing list](https://lists.cncf.io/g/cncf-flux-dev)
- [Our blog](https://github.com/fluxcd/website/issues/27) (WIP)
- Slack (likely more to follow)
  + [#flux](https://cloud-native.slack.com/archives/flux)
  + [#gitops](https://kubernetes.slack.com/archives/gitops)
- Social media
  + [LinkedIn](https://www.linkedin.com/groups/8985374/)
  + Twitter (WIP)
- [Kubernetes Discuss: Ecosystem forum](https://discuss.kubernetes.io/c/announcements/announcements-eco)
```